scripts/cromwell_status_parser.py

```diff
@@ -40,55 +40,129 @@ def get_workflow_metadata_from_api(workflow_id: str):
         sys.exit(1)
 
 
-def parse_workflow_status_and_outputs(json_data: dict):
+def print_parsed_workflow_summary(
+    wf_id: str,
+    sg_id: str,
+    dataset: str,
+    status_dict: dict,
+    outputs: dict,
+):
+    """Prints the execution status and outputs of the sub-workflows."""
+    print(f"\n{dataset} :: {sg_id} :: {wf_id} :: Workflow Summary:\n")
+    for subworkflow in [
+        'CollectCounts',
+        'CollectSVEvidence',
+        'Scramble',
+        'Whamg',
+        'Manta',
+    ]:
+        print(f"  {subworkflow}:")
+        if not status_dict.get(subworkflow):
+            print("    No attempts found")
+            continue
+
+        execution_status = status_dict[subworkflow]
+        if isinstance(execution_status, dict):
+            print(f"    {len(execution_status)} attempt(s):")
+            for attempt, status in execution_status.items():
+                print(f"      {attempt}: {status}")
+                continue
+        else:
+            print(f"    {execution_status}")
+            continue
+
+        if not outputs.get(subworkflow):
+            print("    No outputs found")
+            continue
+        print("    Outputs:")
+        for key, value in outputs[subworkflow].items():
+            print(f"      {key}: {value}")
+    print()
+
+
+def parse_subworkflow_status_and_outputs(
+    subworkflow: str,
+    attempts: list[dict],
+) -> tuple[dict, dict]:
+    """
+    Parse the status and outputs of a sub-workflow from the Cromwell metadata JSON.
+    """
+    status: dict = {}
+    outputs: dict[str, dict] = {}
+    subworkflow_name = subworkflow.split('.')[-1]
+    if not attempts:
+        status[subworkflow_name] = 'Not Started'
+        return status, outputs
+
+    # The attempts dict is a list of attempts, check each to recover the outputs
+    # and status of the sub-workflow. The final status is the status of the last
+    # attempt, which is the most relevant.
```

Comment:
makes sense, so instead of explicitly checking for a success, then going to that attempt and ignoring the rest, that is implicit - if any attempt succeeds it will always be the last attempt